### PR TITLE
Rename helpers and computed properties

### PR DESCRIPTION
**Pull request by Eptis**
My first pull request for OSS ever, also first time working on an Ember addon, so please give me some feedback. I think what I've done can be improved a lot, but I hope I'm in the right direction.

Issue #37:
This PR basically ads a deprecation for people using the old names for computed properties and template helpers and ads the renamed ones to be more inline with moment.js.

### CODE REVIEW

File: addon/computeds/moment-format.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    var propertyValues = EnumerableUtils.map(args, function (arg) {
+      desc = descriptorFor.call(self, arg);
+
+      if (desc && computed._dependentKeys.indexOf(arg) === -1) {
+        computed.property(arg);
+      }
+
+      return desc ? get(self, arg) : arg;
+    });
```
Comment (jasonmit) [inline addon/computeds/moment-format.js]: Map takes a target argument, so you don't need to close over `self`

File: addon/helpers/moment-format.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import Ember from 'ember';
+import momentjs from 'moment';
+
+var momentFormat;
+
+if (Ember.HTMLBars) {
```
Comment (jasonmit) [inline addon/helpers/moment-format.js]: @stefanpenner can we drop support for Handlebars for future versions?

**Comment by jasonmit** [comment]
Tests are failing and but also needs to be squashed

**Comment by Eptis** [comment]
Tests pass now.

**Comment by stefanpenner** [comment]
although im fine with the new names, Im curious as to the motivation behind removing the existing ones?

**Comment by Eptis** [comment]
The `moment` variable was ambiguous as to what it was doing and conflicted in existing code because the regular `moment` object was already being used. The main motivation was to bring the names more inline with moment.js, i.e. `moment.format`, `moment.duration` and `moment.from`

**Comment by stefanpenner** [comment]
did you know you could rename imports?

```js
import { format as momentFormat, ago as momentAgo } from 'ember-moment/computed';
```

**Comment by Eptis** [comment]
Yeah I noticed when I was doing the PR.  What do you think? Just keep the old names and maybe only adjust moment or just keep it as is?



—
Met vriendelijke groet / Kind regards,


Dennis Tel

**Comment by stefanpenner** [comment]
@jasonmit thoughts?

**Comment by jasonmit** [comment]
I would rename the imports to be consistent with the naming going forward.  Thanks for tackling this :+1:

**Comment by jasonmit** [comment]
Ping @Eptis

**Comment by Eptis** [comment]
So remove the old names as well?

**Comment by jasonmit** [comment]
Closing in favor of https://github.com/stefanpenner/ember-moment/pull/69 since there are conflicts due to recent changes

**Pull request closed without merging**